### DISCUSSION TITLE
Help: New Setup

### DISCUSSION THREAD

**Discussion by pramitbiswas** [James-Yu/LaTeX-Workshop#3139] [Q&A]
### Description
I'm new to LaTeX as well as VSCode. Currently, I'm using TeXstudio and want to try out VSCode for my LaTeX writing. While doing so, few hiccups I faced as described below.

### Log messages

<details>
<summary>LaTeX Workshop Output</summary>
When buildig:

	Found root file from active editor: f:\Writtings\main.tex
	Root file remains unchanged from: f:\Writtings\main.tex.
	ACTIONS command invoked.
	RECIPES command invoked.
	BUILD command invoked.
	Found root file from active editor: f:\Writtings\main.tex
	Root file remains unchanged from: f:\Writtings\main.tex.
	Building root file: f:\Writtings\main.tex
	Build root file f:\Writtings\main.tex
	Recipe step 1: pdflatex, -synctex=1,-interaction=nonstopmode,-file-line-error,f:/Writtings/main
	LaTeX log parsed with 15 messages.
	Recipe step 2: biber, Second v0.2
	Recipe step 3: pdflatex, -synctex=1,-interaction=nonstopmode,-file-line-error,f:/Writtings/main
	LaTeX log parsed with 15 messages.
	Recipe step 4: pdflatex, -synctex=1,-interaction=nonstopmode,-file-line-error,f:/Writtings/main
	LaTeX log parsed with 15 messages.
	Recipe of length 4 finished.
	Successfully built f:\Writtings\main.tex
	No PDF viewer connected for f:\Writtings\main.pdf

After this nothing happen; hence, manually opening 'view LaTex pdf'

	ACTIONS command invoked.
	VIEW command invoked.
	Found root file from active editor: f:\Writtings\main.tex
	Root file remains unchanged from: f:\Writtings\main.tex.
	No PDF viewer connected for f:\Writtings\main.pdf
	Serving PDF file at http://127.0.0.1:56813/viewer.html?file=/pdf:f%253A%255CIITP%2520PhD%255CProject%255CWrittings%255CJournal%255CSecond%255CSecond%2520v0.2.pdf
	Open PDF tab for f:\Writtings\main.pdf
	Preview PDF file: f:\Writtings\main.pdf
	Executing synctex with args view,-i,1460:19:f:\Writtings\main.tex,-o,f:\Writtings\main.pdf
	Cannot synctex: spawn synctex ENOENT, 
	Try root file in root directory: f:\Writtings\main v0.0.tex
	Parsing f:\Writtings\main v0.0.tex
	Parsing f:\Writtings\FigTab\n7n.tex
	Try root file in root directory: f:\Writtings\main v0.0s.tex
	Parsing f:\Writtings\main v0.0s.tex
	Parsing f:\Writtings\FigTab\n7n.tex
	Try root file in root directory: f:\Writtings\main v0.1.tex
	Parsing f:\Writtings\main v0.1.tex
	Parsing f:\Writtings\FigTab\n7n.tex
	Try root file in root directory: f:\Writtings\main.tex
	Parsing f:\Writtings\main.tex
	Parsing f:\Writtings\FigTab\n7n.tex

</details>

<details>
<summary>Developer Tools Console</summary>

Nothing suspicious.

</details>

### How to reproduce

OS: Windows 10 x64
MikTeX
VSCode+LaTeX Workshop 

My settings are:

	"latex-workshop.latex.recipes": [
        {
            "name": "pdflatex->biber->pdflatex*2",
            "tools": [
                "pdflatex",
                "biber",
                "pdflatex",
                "pdflatex"
            ]
        }
    ],
    "latex-workshop.latex.tools": [
        {
            "name": "pdflatex",
            "command": "pdflatex",
            "args": [
                "-synctex=1",
                "-interaction=nonstopmode",
                "-file-line-error",
                "%DOC%"
            ]
        },
        {
            "name": "bibtex",
            "command": "bibtex",
            "args": [
                "%DOCFILE%"
            ]
        },
        {
            "name": "biber",
            "command": "biber",
            "args": [
              "%DOCFILE%"
            ]
        },
        {
            "name": "viewPDF",
            "command": "viewPDF",
            "args": [
              "%DOCFILE%"
            ]
        }
    ],
    "latex-workshop.synctex.afterBuild.enabled": true,
    "latex-workshop.view.pdf.viewer": "tab"

### Expected behavior

I want few similar workflow like TeXstudio, they are: i) whenever I run pdflatex and all, it should show me generated pdf in the right side automatically after the finish of compilation, ii) should highlight momentarily the last portion I edited in latex, iii) I can go pdf to latex and vice-versa for selected area.

### Actual behavior

After the successful build pdf does not shows up automatically, I have to open the pdf manually.
After opening the pdf manually, pdf to latex (in TeXstudio it is: right click on the area I want to go, then 'Go to source') and vice-versa (in TeXstudio it is: right click on the area I want to go, then 'Go to pdf') not working.

### Additional Information
Is it possible? Am I missing anything? Any better way?
Thank you very much for enabling a way to write and build LaTeX in VS Code.

**Comment by jlelong** [comment]
pdf to latex and vice-versa works in LaTeX-Workshop and relies on synctex.
- `ctrl + click` in the `.pdf` file to jump to the `.tex` file
- Right click in the `.tex` and choose **SyncTeX from cursor** to jump to the `.pdf` file.

**Comment by pramitbiswas** [comment]
i) generated pdf not showing automatically.
Steps mentioned above didn't work, got the following output:
ii) Cannot reverse synctex: spawn synctex ENOENT.
iii) Cannot synctex: spawn synctex ENOENT.

**Comment by James-Yu** [comment]
ENOENT means your LaTeX distribution does not include synctex.
